lib/routes/javdb/utils.js

```diff
@@ -1,28 +1,27 @@
 const got = require('@/utils/got');
 const cheerio = require('cheerio');
 
-const rootUrl = 'https://javdb.com';
+const rootUrl = 'https://javdb7.com';
```

Comment:
`https://javdb.com` is still accessible. Why changing the domain here?

Reply:
`javdb.com` returns a 403 code. However, `javdb7.com`, which an enthusiastic RSSHub user mentioned in an email, works fine. 😃